packages/lib/models/utils/getCanBeCollapsedFolderIds.ts

```diff
@@ -15,6 +16,18 @@ export default (folders: FolderEntity[]) => {
 		}
 	};
 
+	// Future proofing: if TrashFolder is already in canBeCollapsedIds do not add it again.
+	if (!(getTrashFolderId() in canBeCollapsedIds)) {
+
+		// Currently only deleted folders result in a collapsable Trash
+		for (const folder of folders) {
+			if (folder.deleted_time) {
```

Comment:
This new loop iterates through `folders` while `processTree` loops through both `folders` and child folders.

Is trash correctly marked as collapsible if a subfolder is deleted (but no top-level folders are)?